File: go/level_1/문자열_다루기_기본.go
func solution(s string) bool {
    
	if len(s) != 4 && len(s) != 6 {
			return false;
	}
	
	byteSlice := []byte(s)
	
	for _, value := range byteSlice {
			isNumberCondition := 48 <= value && value <= 57
			
			if !isNumberCondition {
					return false
			}
	}
	
	return true;
}

File: js/level_1/문자열_다루기_기본.js
function solution(s) {
  if (s.length !== 4 || s.length !== 6) return false;

  const splited = s.split("");

  for (value of splited) {
    const isNumberCondition =
      48 <= value.charCodeAt() && value.charCodeAt() <= 57;

    if (!isNumberCondition) return false;
  }

  return true;
}
